specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -3983,6 +4231,121 @@
       },
       "description": "Filters limit replication to a subset of blobs within the storage account. A logical OR is performed on values in the filter. If multiple filters are defined, a logical AND is performed on all filters."
     },
+    "LocalUsers": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalUser"
+          },
+          "description": "The local users associated with the storage account."
+        }
+      },
+      "description": "List storage account local users."
+    },
+    "LocalUser": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LocalUserProperties",
+          "x-ms-client-flatten": true,
+          "description": "Returns the storage account local user properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The local user associated with the storage accounts."
+    },
+    "LocalUserProperties": {
+      "properties": {
+        "permissionScopes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PermissionScope"
+          },
+          "description": "The permission scopes of the local user."
+        },
+        "homeDirectory": {
+          "type": "string",
+          "description": "Optional, local user home directory."
+        },
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Auto generated by the server for SMB."
+        },
+        "sharedKey": {
+          "$ref": "#/definitions/SharedKey"
+        },
+        "hasSharedKey": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Indicates whether shared key exists."
+        },
+        "hasSshKey": {
+          "readOnly": true,
+          "type": "boolean",
+          "description": "Indicates whether ssh key exists."
+        }
+      },
+      "description": "The Storage Account Local User properties."
+    },
+    "LocalUserKeys": {
+      "properties": {
+        "sshAuthorizedKeys": {
+          "$ref": "#/definitions/SshAuthorizedKeys"
+        },
+        "sharedKey": {
+          "$ref": "#/definitions/SharedKey"
+        }
+      },
+      "description": "The Storage Account Local User keys."
+    },
+    "PermissionScope": {
+      "properties": {
+        "permissions": {
+          "type": "string",
+          "description": "The permissions for the local user. Possible values include: Read (r), Write (w), Delete (d), List (l), and Create (c)."
+        },
+        "service": {
+          "type": "string",
+          "description": "The service used by the local user, e.g. blob, file."
```

Comment:
should this be enum?

Reply:
is it possible to specify multiple services in one permission scope?

Reply:
what would be the difference in terms of the user experience? Today we only support blob and file. It is fine to structure it as an enum. Though it gives us more flexibility to add another new resource type. 
There can be only one service in one permission scope.

Reply:
I see. With enum type, CLI would be easy to get all enum values for customer usage guide.